src/main/java/org/veupathdb/service/eda/ds/plugin/abundance/AbundanceBoxplotPlugin.java

```diff
@@ -55,7 +55,7 @@ protected Class<AbundanceComputeConfig> getComputeSpecClass() {
   @Override
   public ConstraintSpec getConstraintSpec() {
     return new ConstraintSpec()
-      .dependencyOrder("overlayVariable", "facetVariable")
+      .dependencyOrder(new String[] {"overlayVariable", "facetVariable"})
```

Comment:
@d-callan If you're willing to go back to all these, you can get rid of some boilerplate like this:
```
  String[][] blah = new String[][] {
    {"a","b"}, {"c","d"}
  };
```